frontend/src/pages/project/Settings/Danger.vue

```diff
@@ -220,8 +220,15 @@ export default {
             this.changeStack(this.project.stack.replacedBy)
         },
         exportProject (parts) {
+            console.log('export project')
             // call projectApi to generate zipped json
             projectApi.exportProject(this.project.id, parts)
+                .then(() => {
+                    alerts.emit('Project export has started successfully.', 'confirmation')
```

Comment:
Export is technically correct, but we're promoting to a new environment. Would be great if it's along the lines of:

"Promoting the current version of <Project Name> to <Project target name>"